grader/settings.py

```diff
@@ -180,10 +172,6 @@
 # Grading action scripts.
 PREPARE_SCRIPT = join(BASE_DIR, "scripts/prepare.sh")
 GITCLONE_SCRIPT = join(BASE_DIR, "scripts/gitclone.sh")
```

Comment:
Shouldn't you also remove `PREPARE_SCRIPT` and `GITCLONE_SCRIPT` since they are only used in the chroot sandbox?

Reply:
Done!